src/storage/mod.rs

```diff
@@ -276,72 +288,60 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
     /// Only writes that are committed before their respective `start_ts` are visible.
     pub fn batch_get_command(
         &self,
-        gets: Vec<PointGetCommand>,
+        requests: Vec<GetRequest>,
     ) -> impl Future<Item = Vec<Result<Option<Vec<u8>>>>, Error = Error> {
         const CMD: CommandKind = CommandKind::batch_get_command;
         // all requests in a batch have the same region, epoch, term, replica_read
-        let ctx = gets[0].ctx.clone();
-        let priority = ctx.get_priority();
-        let priority_tag = get_priority_tag(priority);
+        let priority = CommandPri::Normal;
```

Comment:
Why not get priorities from `requests`?

Reply:
Because we only batch requests of which priority is `Normal` together.